mob

```diff
@@ -56,6 +56,7 @@ import grp
 import os
 import pathlib
 import platform
+import psutil
```

Comment:
not sure if we want to depend on this.  I had it installed on my machine. I don't recall intentionally adding it